content/zh-cn/blog/_posts/2018-07-12-resizing-persistent-volumes-using-kubernetes.md

```diff
@@ -0,0 +1,150 @@
+Kubernetes 会在引用数据卷的 Pod 重新启动时自动处理这个问题。
+
+网络附加文件系统（如 Glusterfs 和 Azure File）可以被扩展，而不需要重新启动引用的 Pod，因为这些系统不需要特殊的文件系统扩展。
+
+文件系统扩展必须通过终止使用该卷的 Pod 来触发。更具体地说：
+
+* 编辑 PVC 以请求更多的空间。
+* 一旦底层卷被存储提供商扩展后， PersistentVolume 对象将反映更新的大小，PVC 会有 `FileSystemResizePending`  状态。
+
+你可以通过运行 `kubectl get pvc <pvc_name> -o yaml` 来验证这一点。
+
+```
+~> kubectl get pvc myclaim -o yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim
+  namespace: default
+  uid: 02d4aa83-83cd-11e8-909d-42010af00004
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 14Gi
+  storageClassName: standard
+  volumeName: pvc-xxx
+status:
+  capacity:
+    storage: 9G
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: 2018-07-11T14:51:10Z
+    message: Waiting for user to (re-)start a pod to finish file system resize of
+      volume on node.
+    status: "True"
+    type: FileSystemResizePending
+  phase: Bound
+```
+<!--
+* Once the PVC has the condition `FileSystemResizePending` then pod that uses the PVC can be restarted to finish file system resizing on the node. Restart can be achieved by deleting and recreating the pod or by scaling down the deployment and then scaling it up again.
+* Once file system resizing is done, the PVC will automatically be updated to reflect new size.
+
+Any errors encountered while expanding file system should be available as events on pod.
+-->
+* 一旦 PVC 具有 `FileSystemResizePending` 状态 ，就可以重启使用该 PVC 的 Pod 以完成节点上的文件系统大小调整。重新启动可以通过删除并重新创建 Pod，或者通过缩小 Deployment 规模然后再次扩大规模来实现。
+* 一旦文件系统的大小调整完成，PVC 将自动更新以展现新的大小。
+
+在扩展文件系统时遇到的任何错误都应作为 Pod 的事件而存在。
+
+<!--
+## Online File System Expansion
+
+Kubernetes v1.11 also introduces an alpha feature called online file system expansion. This feature enables file system expansion while a volume is still in-use by a pod. Because this feature is alpha, it requires enabling the feature gate, `ExpandInUsePersistentVolumes`. It is supported by the in-tree volume plugins GCE-PD, AWS-EBS, Cinder, and Ceph RBD. When this feature is enabled, pod referencing the resized volume do not need to be restarted. Instead, the file system will automatically be resized while in use as part of volume expansion. File system expansion does not happen until a pod references the resized volume, so if no pods referencing the volume are running file system expansion will not happen.
+-->
+## 在线文件系统扩展
+
+Kubernetes v1.11 里还引入了一个名为在线文件系统扩展的 Alpha 功能。
+该功能可以让一个正在被 Pod 使用的卷进行文件系统扩展。因为这个功能是 Alpha 阶段，所以它需要启用特性门控 `ExpandInUsePersistentVolumes`。
+树内卷插件 GCE-PD、AWS-EBS、Cinder 和 Ceph RBD 都支持该功能。当这个功能被启用时，引用调整后的卷的 Pod 不需要被重新启动。
+相反，在使用中文件系统将作为卷扩展的一部分自动调整大小。
+文件系统的扩展是在一个 Pod 引用调整后的卷时才发生的，所以如果没有引用卷的 Pod 在运行，文件系统的扩展就不会发生。
+
+<!--
+## How can I learn more?
+
+Check out additional documentation on this feature here: http://k8s.io/docs/concepts/storage/persistent-volumes.
+-->
+## 更多信息
+
+在这里查看有关这一功能的其他文件：http://k8s.io/docs/concepts/storage/persistent-volumes。
```

Comment:
```suggestion
在这里查看有关这一特性的其他文档：https://kubernetes.io/zh-cn/docs/concepts/storage/persistent-volumes/。
```

Reply:
Thank you for the review, I have made some adjustments.